chart/templates/_helpers.tpl

```diff
@@ -99,3 +99,14 @@ true
 {{- fail "Invalid value for .Values.global.cattle.psp.enabled - must be a bool of true, false, or \"\"" -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Helper function to validate agentTLSMode.
+*/}}
+{{- define "rancher.agent_tls_mode" -}}
```

Comment:
The translation here shouldn't be necessary - both the setting and the chart value should use strings and not booleans.